eslint.config.mjs

```
@@ -1,2 +1,14 @@
 import { config } from '@wcpos/eslint-config';
-export default config;
\ No newline at end of file
+export default [
+	...config,
```

ESLint

[error] Replace `↹` with `··`

(prettier/prettier)